packages/drivers/odsp-driver-definitions/package.json

```diff
@@ -35,6 +35,7 @@
   "devDependencies": {
     "@fluidframework/build-common": "^0.22.0",
     "@fluidframework/eslint-config-fluid": "^0.23.0",
+    "@fluidframework/protocol-definitions": "^0.1024.0",
```

Comment:
'--symlink:full' uncovered three packages that forgot to declare dependencies on protocol-definitions.

Reply:
any clue why it's not a build break?